File: .net & mono/Weed3Demo/config/CacheUtil.cs
using Noear.Weed.Cache;
using Weed3Demo.config;

namespace Weed3Demo {
    public class CacheUtil {
        ICacheServiceEx data = new AspCache("data", 60);
        ICacheServiceEx text = new MemCache("text", 60, "127.0.0.0:8001");
        ICacheServiceEx xxxx = new LocalCache(keyHeader: "xxxx", defSeconds: 60);
    }
}

File: _test/src/main/java/weed3demo/config/CacheUtil.java
package weed3demo.config;

import org.noear.weed.cache.ICacheServiceEx;
import org.noear.weed.cache.LocalCache;
import org.noear.weed.cache.SecondCache;
import org.noear.weed.cache.memcached.MemCache;

/**
 * Created by noear on 2017/7/22.
 */
public class CacheUtil {
    public static boolean isUsingCache;

    ICacheServiceEx cache1 = new MemCache("text", 60, "127.0.0.0:8001",null,null);
    ICacheServiceEx cache2 = new LocalCache("xxxx", 60);

    //二级缓存
    ICacheServiceEx cachex = new SecondCache(cache1,cache2);


}
